Projects/hCore/Sources/Previews.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct ScreenPreview<Screen: View>: View {
+    var screen: Screen
+
+    let colorScheme: ColorScheme
+
+    var body: some View {
+        ForEach(devices) { device in
+            NavigationView {
```

Comment:
Maybe we should use the JourneyPreviewer here to make sure the previews behave exactly as they would in production 🤔